packages/flutter_hooks/test/use_listenable_test.dart

```diff
@@ -72,6 +72,14 @@ void main() {
     expect(listenable.hasListeners, false);
 
     listenable.dispose();
+    listenable = null;
```

Comment:
Could you make a separate test instead of reusing an existing one? Tests should check only one thing at a time

Reply:
It makes sense. I pushed for that.